src/pbt/__init__.py

```diff
@@ -81,19 +82,108 @@ def validate(path, treat_warnings_as_errors):
 )
 @click.option("--skip-builds", is_flag=True, default=False, help="Flag to skip building Pipelines")
 def deploy(
-    path,
-    dependent_projects_path,
-    release_version,
-    project_id,
-    prophecy_url,
-    fabric_ids,
-    job_ids,
-    skip_builds,
+        path,
+        dependent_projects_path,
+        release_version,
+        project_id,
+        prophecy_url,
+        fabric_ids,
+        job_ids,
+        skip_builds,
 ):
     pbt = ProphecyBuildTool(path, dependent_projects_path, release_version, project_id, prophecy_url)
     pbt.deploy(fabric_ids=fabric_ids, skip_builds=skip_builds, job_ids=job_ids)
 
 
+@cli.command()
+@click.option(
+    "--path",
+    help="Path to the directory containing the pbt_project.yml file",
+    required=True,
+)
+@click.option(
+    "--state-config-path",
+    help="Path to the yaml file containing the state configuration",
+    required=True,
+)
+@click.option(
+    "--project-id",
+    help="Prophecy project id to deploy",
+    required=True,
+)
+@click.option(
+    "--release-version",
+    help="Release version to be used during deployments",
+    required=True,
+)
+
+@click.option(
+    "--pipeline-ids",
+    help="pipeline-ids to be build and upload",
+    required=False,
+)
+def build_v2(path, state_config_path, project_id, release_version, pipeline_ids):
+    pbt = PBTCli(path, state_config_path, project_id, release_version)
+    pbt.build(pipeline_ids)
+
+
+@cli.command()
+@click.option(
+    "--path",
+    help="Path to the directory containing the pbt_project.yml file",
+    required=True,
+)
+@click.option(
+    "--state-config-path",
+    help="Path to the yaml file containing the state configuration",
+    required=True,
+)
+@click.option(
+    "--project-id",
+    help="Prophecy project id to deploy",
+    required=True,
+)
+@click.option(
+    "--release-version",
+    help="Release version to be used during deployments",
+    required=True,
+)
+def headers_v2(path, project_id: str, release_version: str, state_config_path: str,):
+    pbt = PBTCli(path, state_config_path, project_id, release_version)
+    pbt.headers()
+
+
+@cli.command()
+@click.option(
+    "--path",
+    help="Path to the directory containing the pbt_project.yml file",
+    required=True,
+)
```

Comment:
n00b question, why do we need to repeat the same click options multiple times? I mean can we register the options once & use them multiple times?